content/hardware/05.pro-solutions/solutions-and-kits/opta/tutorials/getting-started/content.md

```diff
@@ -104,11 +104,11 @@ void loop() {
 
 ### Configuring the Programmable Button on the Opta™
 
-Opta™ has a programmable button, shown in the image below and identified as USER. It can be programmed using the Arduino IDE to fit your needs. To show how simple is to use it, let's create a sketch and program the button as a trigger to modify the status of the STATUS LEDs.
+Opta™ has a programmable button, shown in the image below and identified as USER. It can be programmed using the Arduino IDE to fit your needs. To show how simple it is to use, let's create a sketch and program the button as a trigger to modify the status of the STATUS LEDs.
```

Comment:
@Jorgetrujilloroman can you give me more context ?

for me both versions are valid and I prefer to have the "it" at the end of the sentence